From 5a4d374315639c61bed10cefd699d5c76b22f5f2 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Fri, 19 Aug 2022 16:34:07 +0200
Subject: [PATCH] Don't register dummy org.freedesktop.secrets service when api
 is disabled

It prevents other programs from claiming that service

BUG: 458069
---
 src/runtime/kwalletd/kwalletd.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/runtime/kwalletd/kwalletd.cpp b/src/runtime/kwalletd/kwalletd.cpp
index 5de735bd..d10ce59e 100644
--- a/src/runtime/kwalletd/kwalletd.cpp
+++ b/src/runtime/kwalletd/kwalletd.cpp
@@ -168,9 +168,6 @@ KWalletD::KWalletD()
 
     if (cfgSecrets.readEntry<bool>("apiEnabled", true)) {
         _fdoService.reset(new KWalletFreedesktopService(this));
-    } else {
-        /* Do not keep dbus-daemon waiting for the org.freedesktop.secrets by registering the dummy-service */
-        KWalletFreedesktopService(nullptr);
     }
 }
 
-- 
GitLab

